pkgs/by-name/si/signal-desktop/generic.nix

```
@@ -5,6 +5,8 @@
   autoPatchelfHook,
   noto-fonts-color-emoji,
   dpkg,
+  rpm,
```

See the comment about libarchive.
```suggestion
  libarchive,
```